designsafe/apps/auth/tasks.py

```diff
@@ -17,76 +21,68 @@
 logger = logging.getLogger(__name__)
 
 
+# TODOV3 replace following by importing mkdir from server/portal/libs/agave/operations.py
+# TODOV3 when added during https://tacc-main.atlassian.net/browse/DES-2703
+def mkdir(client, system, path, dir_name):
+    client.files.mkdir(systemId=system, path=path_input)
+    return {"result": "OK"}
+
+
 @shared_task(default_retry_delay=30, max_retries=3)
-def check_or_create_agave_home_dir(username, systemId):
+def check_or_configure_system_and_user_directory(username, system_id, create_credentials):
```

Comment:
Maybe `path` should be a param. Since `path` for HOME is username, but `path` for WORK is going to be `/work/1234/username` (prefix gained via TAS)

Reply:
added as param but always `username` for the time being.